File: app/src/main/java/rekha/com/locationtracker/view/BaseActivity.java
package rekha.com.locationtracker.view;

import android.support.v7.app.AppCompatActivity;
import android.widget.Toast;

public class BaseActivity extends AppCompatActivity {
    public void showMessage(String msg) {
        Toast.makeText(this, msg, Toast.LENGTH_LONG).show();
    }
}

File: app/src/main/java/rekha/com/locationtracker/view/BaseActivity.kt
package rekha.com.locationtracker.view

import android.support.v7.app.AppCompatActivity
import android.widget.Toast

open class BaseActivity : AppCompatActivity() {
    fun showMessage(msg: String) {
        Toast.makeText(this, msg, Toast.LENGTH_LONG).show()
    }
}
